apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/commons/QueryUtils.java

```diff
@@ -222,33 +224,14 @@ public static boolean isInRange(Instant lastUpdated, DateRangeParam range) {
     return true;
   }
 
-  /**
-   * Convert an integer value into claims data available BitSet.
-   *
-   * @param maskVal integer value suitable for decomposing into a Claims BitSet.
-   * @return {@link BitSet} denoting which claims have data.
-   */
-  public static BitSet convertClaimsBitmaskValue(int maskVal) {
-    BitSet rslt = new BitSet(maskVal);
-    rslt.set(CARRIER_HAS_DATA, (maskVal & V_CARRIER_HAS_DATA) != 0);
-    rslt.set(INPATIENT_HAS_DATA, (maskVal & V_INPATIENT_HAS_DATA) != 0);
-    rslt.set(OUTPATIENT_HAS_DATA, (maskVal & V_OUTPATIENT_HAS_DATA) != 0);
-    rslt.set(SNF_HAS_DATA, (maskVal & V_SNF_HAS_DATA) != 0);
-    rslt.set(DME_HAS_DATA, (maskVal & V_DME_HAS_DATA) != 0);
-    rslt.set(HHA_HAS_DATA, (maskVal & V_HHA_HAS_DATA) != 0);
-    rslt.set(HOSPICE_HAS_DATA, (maskVal & V_HOSPICE_HAS_DATA) != 0);
-    rslt.set(PART_D_HAS_DATA, (maskVal & V_PART_D_HAS_DATA) != 0);
-    return rslt;
-  }
-
   /**
    * Query database to determine which claim types have data for the specified beneficiary.
    *
    * @param entityManager {@link EntityManager} used to query database.
    * @param beneficiaryId used to identify the Beneficiary to check claims for.
-   * @return {@link BitSet} denoting which claims have data.
+   * @return {@link int} bitmask denoting which claims have data.
```

Comment:
Does `{@link int}` actually do anything?  Id just use the work `int` or `integer`.